### ISSUE TITLE
Suggestion: Allow importing Alyle UI in sub-modules

### ISSUE THREAD

**Issue by uplink42** [A-l-y-l-e/Alyle-UI#87]
#### Bug, feature request, or proposal:
feature request

#### What is the expected behavior?
I've recently discovered Alyle UI and it's a great library with many useful components. However from what I understand from the documentation you are only able to import the LyThemeModule in the root module. This adds a significant weight to the main bundle size in my project, especially because I only actually use the Alyle components inside lazy-loaded modules (i.e the admin sections).

So basically, the main bundle is forced to include **Alyle and chroma-js** but most of my app doesn't actually use it. It's a bit of a letdown, considering the homepage is a public-facing app where speed and loading times are important. For example, with Angular Material Design I only load the components for the admins section and they don't put any extra weight on the initial bundle.

I hope you can consider this sometime. Great work btw.

**Comment by Enlcxx** [comment]
Yes, this will definitely have support for lazy-loading, so it will be available for version 3. It is very likely that the API can change.
